core/app/models/spree/store.rb

```diff
@@ -129,7 +146,9 @@ def default_menu(location)
     end
 
     def supported_currencies_list
-      @supported_currencies_list ||= (read_attribute(:supported_currencies).to_s.split(',') << default_currency).sort.map(&:to_s).map do |code|
+      ensure_supported_currencies
```

Comment:
I'm not sure about this one, is it related to the changes in store translations?

Reply:
Yes, and once again this change is not needed because we are no longer translating supported_currencies and default_currency. Will remove 😎